### PR TITLE
Release 1.7.0: update changelog and shared to v170

### PR DESCRIPTION
**Pull request by KievDevel**

### CODE REVIEW

**Review by boundlesscalm — APPROVED** [review]
Looks fine from my side but I think @greg-fer may have some comments about the changelog if I remember some conventions previously used though I may be wrong.
Do change the label to be "Docs required" in case greg hasn't looked over it, please.

**Comment by KievDevel** [comment]
@boundlesscalm was copied from PPK changelog (same change) from approved PR by Greg but will doublecheck with him

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.7.0 - 2024-04-08
+
+### Changed
+
+-   Moved feedback tab to a dialog which can be opened by going to the about tab
+    and click **Give Feedback**.
```
Comment (greg-fer) [inline Changelog.md:6]:
```suggestion
-   Moved the Feedback tab to the **Give Feedback** dialog in the About tab.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Let's be short and concise :D

File: Changelog.md (hunk 1 above)
Comment (KievDevel) [reply Changelog.md:6]: What about "Moved the Feedback tab into the **Give Feedback** dialog in the About tab." ?
make concise back again :D

Comment (greg-fer) [reply Changelog.md:6]: I shortened my proposal to `` Integrated the functionality of the Feedback tab in the **Give Feedback** dialog in the About tab.``
The reason why I didn't like the "Moved..." proposal was that you didn't move the tab -- there's no tab under Give Feedback -- you just moved the functionality.

**Pull request merged**